packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -322,26 +281,64 @@ export class CheckpointService implements TreeDataProvider<BaseNode> {
             returnString
           ) as QueryOverlayActionIdsSuccessResult;
           if (successResult) {
-            for (const record of successResult.records) {
-              const deleteCommand = new ApexExecutionOverlayActionCommand(
-                undefined,
-                record.Id
+            // If there are things to delete then create the batchRequest
+            if (successResult.records.length > 0) {
+              const requests: BatchRequest[] = [];
+              for (const record of successResult.records) {
+                const request: BatchRequest = {
+                  method: RestHttpMethodEnum.Delete,
+                  url:
+                    'services/data/v43.0/tooling/sobjects/ApexExecutionOverlayAction/' +
```

Comment:
I thought you had a constant for this?

Reply:
You're right, I have OVERLAY_ACTION_DELETE_URL. Done.